Python/Product/PythonTools/PythonTools/Environments/AddCondaEnvironmentControl.xaml

```diff
@@ -162,13 +163,13 @@
                 Text="{x:Static common:Strings.CondaNotFoundError}"
                 TextWrapping="Wrap"/>
         </Grid>
-        <ItemsControl 
+        <ScrollViewer VerticalScrollBarVisibility="Auto">
```

Comment:
Moved the scroll viewer out of just the 'left' hand side controls. Now it will allow scrolling in those cases where the screen is too thin.